Ryujinx.HLE/HOS/Services/ServerBase.cs

```diff
@@ -78,7 +78,15 @@ public void AddSessionObj(KServerSession serverSession, IpcService obj)
         public void AddSessionObj(int serverSessionHandle, IpcService obj)
         {
             _sessionHandles.Add(serverSessionHandle);
-            _sessions.Add(serverSessionHandle, obj);
+            if(!_sessions.ContainsKey(serverSessionHandle))
```

Comment:
```suggestion
            if (!_sessions.ContainsKey(serverSessionHandle))
```

Imo this looks like more as a hack than a real fix :/

Reply:
I understand and would love some help in reaching a better solution if possible. This PR avoided crashes and, because I remove sessions when they collide in key, it remains viable to run. That's why it's submitted at all, but I know it deserves polish.